test/models/partner_test.rb

```diff
@@ -420,7 +420,7 @@ class PartnerTest < ActiveSupport::TestCase
 
   test 'partnership_admin can create a partner that is part of their partnership' do
     pa = create(:partnership_admin)
-    partner = create(:partner, :accessed_by_user => pa, :tags => [pa.tags.first])
+    partner = create(:partner, :accessed_by_user => pa, :partnerships => [pa.partnerships.first])
```

Comment:
So this hasn't been working since https://github.com/geeksforsocialchange/PlaceCal/pull/2378 when we did the tag refector?

Reply:
I think so.

Reply:
There are no system tests for creating a new partner as far as I can tell. I think the current test suite separation of concerns needs some attention.